src/operator/nn/dropout-inl.h

```diff
@@ -367,29 +475,54 @@ class DropoutOp {
   /*! \brief Dropout mode */
   dropout::DropoutOpMode mode_;
   TShape axes_;
+  bool dropout_passthrough_;
+#if MXNET_USE_CUDNN_DROPOUT
+  bool cudnn_off_;
+  Context ctx_;
+  cudnnDataType_t dtype_;
+  cudnnDropoutDescriptor_t dropout_desc_;
+  uint64_t seed_ = 17 + rand() % 4096;  // NOLINT(runtime/threadsafe_fn)
```

Comment:
Does this affect reproducibility?

Reply:
it shouldn't as `rand()` is controlled by the global seed too.